contracts/test/ERC20.t.sol

```diff
@@ -303,58 +310,56 @@ contract ERC20PermitTest is TestUtils {
         // Any other value should fail.
         // If v is 27, then 28 should make it past the MALLEABLE require, but should result in an invalid signature, and vice versa when v is 28.
-        ( uint8 v, bytes32 r, bytes32 s ) = _getValidPermitSignature(amount, owner, skOwner, deadline);
+        ( uint8 v, bytes32 r, bytes32 s ) = _getValidPermitSignature(amount, _owner, _skOwner, _deadline);
 
         for (uint8 i; i <= type(uint8).max; i++) {
             if (i == type(uint8).max) {
                 break;
             } else if (i != 27 && i != 28) {
-                vm.expectRevert(bytes("ERC20:P:MALLEABLE"));
+                vm.expectRevert("ERC20:P:MALLEABLE");
             } else {
-                if (i == v) {
-                    continue;
-                } else {
-                    // Should get past the Malleable require check as 27 or 28 are valid values for s.
-                    vm.expectRevert(bytes("ERC20:P:INVALID_SIGNATURE"));
-                }
+                if (i == v) continue;
+
+                // Should get past the Malleable require check as 27 or 28 are valid values for s.
+                vm.expectRevert("ERC20:P:INVALID_SIGNATURE");
             }
-            user.erc20_permit(address(token), owner, spender, amount, deadline, i, r, s);
+
+            _user.erc20_permit(address(_token), _owner, _spender, amount, _deadline, i, r, s);
         }
 
-        user.erc20_permit(address(token), owner, spender, amount, deadline, v, r, s);
+        _user.erc20_permit(address(_token), _owner, _spender, amount, _deadline, v, r, s);
     }
 
     // Returns an ERC-2612 `permit` digest for the `owner` to sign
-    function _getDigest(address owner_, address spender_, uint256 value_, uint256 nonce_, uint256 deadline_) internal view returns (bytes32) {
+    function _getDigest(address owner_, address spender_, uint256 amount_, uint256 nonce_, uint256 deadline_) internal view returns (bytes32 digest_) {
         return keccak256(
             abi.encodePacked(
                 '\x19\x01',
-                token.DOMAIN_SEPARATOR(),
-                keccak256(abi.encode(token.PERMIT_TYPEHASH(), owner_, spender_, value_, nonce_, deadline_))
+                _token.DOMAIN_SEPARATOR(),
+                keccak256(abi.encode(_token.PERMIT_TYPEHASH(), owner_, spender_, amount_, nonce_, deadline_))
             )
         );
     }
 
     // Returns a valid `permit` signature signed by this contract's `owner` address
-    function _getValidPermitSignature(uint256 value_, address owner_, uint256 ownerSk_, uint256 deadline_) internal returns (uint8 v_, bytes32 r_, bytes32 s_) {
-        bytes32 digest = _getDigest(owner_, spender, value_, nonce, deadline_);
-        ( uint8 v, bytes32 r, bytes32 s ) = vm.sign(ownerSk_, digest);
-        return (v, r, s);
+    function _getValidPermitSignature(uint256 amount_, address owner_, uint256 ownerSk_, uint256 deadline_) internal returns (uint8 v_, bytes32 r_, bytes32 s_) {
```

Comment:
Maybe reorder parameters based on order in the function call?

Reply:
Yup, but I'll do one slightly better.